skimage/measure/tests/test_simple_metrics.py

```diff
@@ -52,12 +52,12 @@ def test_PSNR_errors():
 def test_NRMSE():
     x = np.ones(4)
     y = np.asarray([0., 2., 2., 2.])
-    assert_equal(normalized_root_mse(y, x, 'mean'), 1 / np.mean(y))
-    assert_equal(normalized_root_mse(y, x, 'Euclidean'), 1 / np.sqrt(3))
-    assert_equal(normalized_root_mse(y, x, 'min-max'), 1 / (y.max() - y.min()))
+    assert_equal(normalized_root_mse(y, x), 1 / np.mean(y))
+    assert_equal(normalized_root_mse(y, x), 1 / np.sqrt(3))
+    assert_equal(normalized_root_mse(y, x), 1 / (y.max() - y.min()))
```

Comment:
I think these need the old `norm_type` keyword instead of removing these arguments.

Reply:
Oops! These were changed automatically by PyCharm during the refactor of the kwarg, but they shouldn't have been using the new API — they should have been testing the existing API! So I've changed them back. I think the new commits should fix the build!